examples/resources/equinix_network_device/example_10.tf

```diff
@@ -0,0 +1,26 @@
+# Create C8000V BYOL device with bandwidth tier information
```

Comment:
When there are multiple examples for a resource or data source, it's better to use more descriptive file names.  In this case, that might be `examples/resources/equinix_network_device/c800v_byol_with_bandwidth_tier.tf`

Reply:
thanks @ctreatma , done

Reply:
renamed, thank you